src/nls/it/strings.js

```diff
@@ -648,7 +648,104 @@ define({
+    "DESCRIPTION_JSLINT_OPTIONS_NOMEN"               : "Vero per consentire penzoloni di sottolineatura in identificatori",
+    "DESCRIPTION_JSLINT_OPTIONS_PASSFAIL"            : "Vero per fermarsi al primo errore",
+    "DESCRIPTION_JSLINT_OPTIONS_PLUSPLUS"            : "Vero per consentire ++ e --",
+    "DESCRIPTION_JSLINT_OPTIONS_REGEXP"              : "Vero per consentire . e [^...]. in RegExp",
+    "DESCRIPTION_JSLINT_OPTIONS_RHINO"               : "Vero se globali Rhino devono essere predefiniti",
+    "DESCRIPTION_JSLINT_OPTIONS_SLOPPY"              : "Vero per consentire di omettere `use strict`",
+    "DESCRIPTION_JSLINT_OPTIONS_STUPID"              : "Vero per consentire la stupidità",
+    "DESCRIPTION_JSLINT_OPTIONS_SUB"                 : "Vero per consentire indicizzazione inefficiente",
+    "DESCRIPTION_JSLINT_OPTIONS_TODO"                : "Vero per consentire commenti Cose da fare",
+    "DESCRIPTION_JSLINT_OPTIONS_UNPARAM"             : "Vero per consentire parametri non utilizzati",
+    "DESCRIPTION_JSLINT_OPTIONS_VARS"                : "Vero per consentire a più di 1 var in una funzione",
+    "DESCRIPTION_JSLINT_OPTIONS_WHITE"               : "Vero per ignorare le regole spazio-bianco",
+    "DESCRIPTION_LANGUAGE"                           : "Impostazioni specifiche della lingua",
+    "DESCRIPTION_LANGUAGE_FILE_EXTENSIONS"           : "Ulteriori mappature di estensione a nome di lingua",
+    "DESCRIPTION_LANGUAGE_FILE_NAMES"                : "Ulteriori mappature da nome di file a nome di lingua",
+    "DESCRIPTION_LINTING_ENABLED"                    : "Vero abilità Code Inspection",
+    "DESCRIPTION_ASYNC_TIMEOUT"                      : "Il tempo in millisecondi dopo la pausa del linter asincrono",
+    "DESCRIPTION_LINTING_PREFER"                     : "Vettore di linter da eseguire prima",
+    "DESCRIPTION_LIVE_DEV_MULTIBROWSER"              : "Vero abilita Esperimento Anteprima Live",
+    "DESCRIPTION_USE_PREFERED_ONLY"                  : "Vero avvia fornitori solo da linting.prefer",
+    "DESCRIPTION_MAX_CODE_HINTS"                     : "Massimo numero di suggerimenti visualizzati alla volta",
+    "DESCRIPTION_PATH"                               : "Impostazioni di percorso specifiche",
+    "DESCRIPTION_PROXY"                              : "L'URL del server proxy utilizzato per l'installazione ampliamento",
+    "DESCRIPTION_SCROLL_PAST_END"                    : "Vero per attivare lo scorrimento oltre la fine del documento",
+    "DESCRIPTION_SHOW_CODE_HINTS"                    : "Falso per disattivare tutti i suggerimenti sul codice",
+    "DESCRIPTION_SHOW_CURSOR_WHEN_SELECTING"         : "Mantiene il cursore lampeggiante visibile quando si dispone di una selezione di testo",
+    "DESCRIPTION_SHOW_LINE_NUMBERS"                  : "Vero per mostrare i numeri di riga in un " grondaia " a sinistra del codice",
```

Comment:
**`true`**` per mostrare i numeri delle righe a sinistra del codice`

(This line also is syntactically invalid)